analyses/archive/seconds_between_shifts.sql

```diff
@@ -0,0 +1,115 @@
+with game_seconds as (
+    select -1 + row_number() over() as seconds
+    from unnest((select split(format("%10800s", ""), '') as h from (select null))) as pos
+    order by seconds
+)
+
+, seconds_between_shifts as (
+    select
+        concat(shifts.shift_id, '_', gs.seconds) as new_shift_id
+        , gs.seconds as game_time_seconds
+        , (gs.seconds - shifts.start_seconds_elapsed) as shift_time_seconds
+        , players.primary_position_abbreviation
+        , case when gs.seconds = shifts.start_seconds_elapsed then true else false end as is_shift_start
+        , case when gs.seconds = shifts.end_seconds_elapsed then true else false end as is_shift_end
+        , shifts.*
+    from `nhl-breakouts.dbt_dom.stg_nhl__shifts` as shifts
+    inner join game_seconds as gs on gs.seconds between shifts.start_seconds_elapsed and shifts.end_seconds_elapsed
+    left join `nhl-breakouts.dbt_dom.d_players` as players on players.player_id = shifts.player_id
+    where 1 = 1
+        and shifts.game_id = 2015021169
+)
+
+, home_player_lists as (
```

Comment:
this is the approach of "grouping" out the player array (by home and away, joining back at the end)... b/c of the sheer size, it will not run if we don't filter on game_id (line 20)... need to think of better approach

Reply:
@gavh3

Reply:
yes